### PR TITLE
Zip code 

### PR DESCRIPTION
**Pull request by glennparham**
I am integrating [this feature](https://www.notion.so/New-Current-Case-Notification-76018a8f2ef74ed8b3bf958c76271008).

I have added zip_code.py.  Trigger the **get_state_stats(zip_code)** function and it will automatically return the properly formatted response as a string.

EXAMPLE RESPONSE: 
Screen Shot 2020-03-20 at 6 31 39 AM

### CODE REVIEW

**Review by Marwan01 — COMMENTED** [review]
Return only state info (comment out county part for now, but leave in code as it might come useful later)

Add number of recovered, and deaths.
Add the yesterday part, and growth rate to the state/country regular search message so it’s conform
I like your formatting where you have the state returned in caps in it’s own line. Do you mind changing the state/country  message to match that too?
So in a nutshell, make the country/state message and the zip message the same.
In the welcome message, add “zip” as follows: [ - Text country/state/“zip” to get the most up to date report.. ]

I LOVE the latest updates from governor. I do find twitter inaccurate tho. I would say since we want to have one small and clean message for the count report, to remove it from that response. However, we have a “news” route, and we could have a different response that will get your governor’s updates, potentially from newsapi or twitter. We could send something like “news” Missouri and you can start getting your governor’s news that way.

**Comment by Marwan01** [comment]
Forget about: Return only state info (comment out county part for now, but leave in code as it might come useful later)

County info is actually awesome when available, and is implemented nicely.

**Comment by AlexAndrei98** [comment]
Glenn, Looks great! I added just a little comment!
Do you think that it would be possible to integrate the growth info and the daily information in the general response message as well? Do you think that would be pretty straight froward to implement?

File: src/twillio_utils.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 from .data_utils import *
 from .responses import *
 from .news import return_news
-
+from .zip_code import get_zip_code_stats
```
Comment (AlexAndrei98) [inline src/twillio_utils.py]: I would remove the `.` from here since it was giving some errors with the deploymebnt

**Review by AlexAndrei98 — CHANGES_REQUESTED** [review]

**Review by AlexAndrei98 — APPROVED** [review]
LGTM let’s add a ticket to add some tests for these cases as well

**Comment by Marwan01** [comment]
Thank you for your contribution!

**Pull request merged**